nephio/optional/fluxcd/rbac.yaml

```diff
@@ -0,0 +1,179 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: critical-pods
+spec:
+  hard:
+    pods: "1000"
+  scopeSelector:
+    matchExpressions:
+    - operator: In
+      scopeName: PriorityClass
+      values:
+      - system-node-critical
+      - system-cluster-critical
```

Comment:
Does this come by default?

Reply:
Yep. Taken directly from the manifests.
https://github.com/fluxcd/flux2/releases/download/v2.4.0/manifests.tar.gz